geode-core/src/test/java/org/apache/geode/internal/statistics/OsStatisticsProviderTest.java

```diff
@@ -56,10 +54,9 @@ public void givenSolarisOs_thenOsStatsAreNotSupported() {
   }
 
   @Test
-  public void givenUnknownOs_thenBuilderThrowsAnException() {
+  public void givenUnknownOs_thenOsStatsAreNotSupported() {
```

Comment:
The exception grabber an its import can be deleted, as there are no tests throwing exceptions:

```
import org.junit.rules.ExpectedException;
...
@Rule
public ExpectedException exceptionGrabber = ExpectedException.none();
```

Reply:
Thanks!